src/line_range.rs

```diff
@@ -47,7 +47,16 @@ impl LineRange {
             }
             2 => {
                 new_range.lower = line_numbers[0].parse()?;
-                new_range.upper = line_numbers[1].parse()?;
+
+                new_range.upper = if line_numbers[1].bytes().next().unwrap() == b'+' {
```

Comment:
I *think* the `.unwrap()` call here is safe (but only due to the checks in lines 33:39), but still: it might be better to avoid it in case the surrounding logic here changes in the future. I think you could simply go with:
```rust
if line_numbers[1].bytes().next() == Some(b'+') {
```
what do you think?

Reply:
I don't like using `.unwrap()` too, but it looked impossible to fail.
Your suggestion is making it even safer, and much better. Thanks.